### PR TITLE
Demo Data

### LINKED ISSUE #1899: Create Demo content for Metacamp
To do the following:

- Hardcode some stuff
- Link (in the frontend only) given Snapshot spaces with given Safe Addresses
  - awaiting the list of Safe addresses and ENS names @parkermccurley 
- If a snapshot space is already set for a given Safe, don't override it

### PR DESCRIPTION
**Pull request by adamgall**
Closes #1899 

Support hardcoded Safe Address / Name / Snapshot ENS data.

Addresses currently implemented:

| dao        | network | address                                      |
| ---------- | ------- | -------------------------------------------- |
| decent     | mainnet | `0xB98d45F9021D71E6Fc30b43FD37FB3b1Bf12c064` |
| coordinape | mainnet | `0xd2786bEc22F4F2500E0dd4E42cE7c9dceBB5Ff93` |

### CODE REVIEW

File: src/hooks/DAO/loaders/useFractalNode.ts [hunk 1]
```diff
@@ -33,29 +35,50 @@ export const useFractalNode = (
...
+            switch (_daoAddress) {
+              case '0xB98d45F9021D71E6Fc30b43FD37FB3b1Bf12c064': {
+                snapshotENS = 'decent-dao.eth';
+                break;
+              }
+              default:
+                break;
+            }
+          }
+        }
```
Comment (adamgall) [inline src/hooks/DAO/loaders/useFractalNode.ts:64]: This is the only new code in this branch, it's a bummer that so much whitespace changed as well.

**Review by tomstuart123 — APPROVED** [review]

**Comment by tomstuart123** [comment]
nice, this works well for me on CB wallet Mobile. 

Thanks for being creative without designs . This is a nice solution

File: src/hooks/DAO/loaders/useFractalNode.ts [hunk 2]
```diff
@@ -33,29 +35,50 @@ export const useFractalNode = (
...
+      if (dao) {
+        let snapshotENS: Maybe<string> | undefined;
+        const { parentAddress, name, proposalTemplatesHash } = dao;
+        snapshotENS = dao.snapshotENS;
+
+        // don't override if already set
+        if (!snapshotENS) {
+          if (networkConfig.chain === mainnet) {
+            switch (_daoAddress) {
+              case '0xB98d45F9021D71E6Fc30b43FD37FB3b1Bf12c064': {
```
[the remark this answers is not in the document]
Comment (adamgall) [reply src/hooks/DAO/loaders/useFractalNode.ts:56]: Haha yeah you don't have much context here.

I'll explain on standup, but the TL;DR is that Parker wants to be able to demo the app to various DAOs at MetaCamp, showing them _their own Snapshot Space hooked up to their own Safe within Decent_.

The only way to do that, without getting those Safes to execute proposals, is to do what I'm doing here. This is just temporary until after Metacamp.

**Comment by tomstuart123** [comment]
Decent and Coordinape work here. Approved

is there still a need for more addresses?

**Review by tomstuart123 — APPROVED** [review]

**Comment by adamgall** [comment]
@tomstuart123 yep, still waiting on more addresses from @parkermccurley for "LexDAO" and "Hydra"

Screenshot 2024-05-29 at 8 06 11 AM

**Comment by adamgall** [comment]
Manually overriding checks and merging this. Will open up another PR after this to revert these changes.

**Pull request merged**